README.md

```diff
@@ -33,6 +33,8 @@ Compared with other graph database solutions, **Nebula Graph** has the following
 
 The Nebula Graph team is glad to announce the Nebula Graph 2.0.0 GA. This is a brand new start with several important updates, and more of them are on the way. Welcome to join us.
 
+After v2.5.0, we use [Nebula repo](https://github.com/vesoft-inc/nebula).
```

Comment:
Do we need to update the paragraph above this one and change "Nebula Graph 2.0.0 GA" to "Nebula Graph 2.5.0"?

Reply:
Yes I think so if we want to keep the Notice of Release section. However, as a rule of thumb, I would not suggest we put such kind of notice in README because the versions are constantly evolving. It is just annoying that you have to update the version number every time there is a new release. People know where to find the latest version of the product. It just should NOT be a necessary part of a README file IMHO.

Reply:
Yes, I deleted this description in the readme of the nebula repo.